pkg/build/nodeimage/const_cni.go

```diff
@@ -20,7 +20,7 @@ package nodeimage
 The default CNI manifest and images are our own tiny kindnet
 */
 
-const kindnetdImage = "docker.io/kindest/kindnetd:v20240813-c6f155d6"
+const kindnetdImage = "docker.io/kindest/kindnetd:v20241007-36f62932"
```

Comment:
we should probably also push a new node image based on this change, you can push to the kind staging registry first so we can test it in CI

Reply:
(the github actions jobs don't build images, they run the default node image, also users won't have an easy time trying the fix without a node image)

Reply:
also nit but: this should be a seperate commit from the source changes. otherwise the git hash is for a commit that doesn't exist.

Reply:
is not a separate commit already?

Reply:
going to sleep https://console.cloud.google.com/gcr/images/k8s-staging-kind/global/node@sha256:3b24a9aaeaafb74c822848ba551d79b21b0b92270d931efb498bca7af01a6dc0/details?inv=1&invt=AbeKYg&tab=pull , I can´t remember how to create the multi arch node image, will take a look tomorrow

Reply:
find it `KUBEROOT=~/src/kubernetes REGISTRY=gcr.io/k8s-staging-kind hack/release/build/push-node.sh`

Reply:
pushed and tested